dom/events/Event-dispatch-click.html

```diff
@@ -241,6 +241,54 @@
   input.dispatchEvent(new MouseEvent("click"));
 }, `disabled radio should fire onclick`);
 
+async_test(t => {
+  const input = document.createElement("input");
+  input.type = "checkbox";
+  input.disabled = true;
+  input.onclick = t.step_func(ev => {
+    assert_true(input.checked);
+    ev.preventDefault();
+    setTimeout(t.step_func_done(() => {
```

Comment:
I think lint will fail on setTimeout usage. queueMicrotask() should suffice, right? Otherwise `t.step_timeout()` would work.